### DISCUSSION TITLE
Adding a menu

### DISCUSSION THREAD

**Discussion by torhus** [torhus/monster-browser#56] [Ideas]
There are some things that could be useful to have access to from within the program. I try to avoid adding clutter, so how about repurposing the Settings button, like this?

![tools-menu](https://user-images.githubusercontent.com/19881605/202331439-0d9bc370-bbf2-4f8f-a40d-d785121762fd.png)

**Comment by The-Gig** [comment]
Seems fine!

**Reply by The-Gig** [reply]
By the way, I mentioned that maybe a menu with some links might have been useful back here: <https://github.com/torhus/monster-browser/issues/32#issue-1406147456> ;-)

**Comment by torhus** [comment]
Seems good?

![Screenshot 2022-11-18 142640](https://user-images.githubusercontent.com/19881605/202715631-560f4463-b51b-46d6-886e-9b65b425e105.png)

**Reply by The-Gig** [reply]
Nice!
Maybe it might include links to show readme/changelog files, and maybe "reset game configuration", being a relatively "dangerous" option, might have been placed deeper, under "settings" window instead, but it's good!
